src/Microsoft.Performance.SDK/Processing/ColumnBuilding/ColumnVariantIdentifier.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Performance.SDK.Processing.ColumnBuilding;
+
+/// <summary>
+///     Represents a identifier for a column variant.
```

Comment:
>column variant [](http://example.com/codeflow?start=38&length=14)

Is a column variant defined in a doc comment somewhere? It seems like something we should define as we use it quite often. If not, would this be a decent place to do so?

Reply:
Added